app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -124,6 +133,7 @@ class HomeActivityTest {
     setUpTestApplicationComponent()
     testCoroutineDispatchers.registerIdlingResource()
     profileTestHelper.initializeProfiles()
+    profileTestHelper.addLongNameProfile()
```

Comment:
Do we need this for all the tests?

Reply:
If not, you can use this before activity launch in the required test case.

Reply:
Shifted this to `initializeProfiles`